### PR TITLE
[tempo-distributed] use old root to get namespace

### PR DESCRIPTION
**Pull request by kaflake**
fix bug from #2723 which produces in #2679.

Namespace-Variable was nil and cause of that its not possible to build with basicauth.

### CODE REVIEW

File: charts/tempo-distributed/templates/gateway/secret-gateway.yaml [hunk 1]
```diff
@@ -1,11 +1,12 @@
+{{- $root := . -}}
 {{- $dict := dict "ctx" . "component" "gateway" -}}
 {{- with .Values.gateway }}
 {{- if and .enabled .basicAuth.enabled (not .basicAuth.existingSecret) }}
 apiVersion: v1
 kind: Secret
 metadata:
   name: {{ include "tempo.resourceName" $dict }}
-  namespace: {{ .Release.Namespace }}
+  namespace: {{ $root.Release.Namespace }}
```
Comment (zanhsieh) [inline charts/tempo-distributed/templates/gateway/secret-gateway.yaml:9]:
```suggestion
  namespace: {{ $root.Release.Namespace | quote }}
```

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Review by zanhsieh — CHANGES_REQUESTED** [review]
@kaflake 
Can you bump the version in Chart.yaml and sign DCO please?

**Comment by kaflake** [comment]
reopen with the new version. Sorry, I'm not working so often in github :/
Hoping this ist now working. Make the changes but do not request a new review.

**Review by zanhsieh — APPROVED** [review]

**Review by kaflake — COMMENTED** [review]
reviewed

**Comment by kaflake** [comment]
What have I to do to finish this pull-request? Do I still a mistake?

**Comment by zanhsieh** [comment]
> What have I to do to finish this pull-request? Do I still a mistake?

Ask other reviewers to see they have different opinions.

**Comment by rlex** [comment]
can we please get it merged? I'm also affected and chart was unusable for me since 1.6.9. Of course i can pin to 1.6.9 and use image.tag but still.

**Review by zalegrala — APPROVED** [review]

**Comment by zalegrala** [comment]
LGTM, the chart needs a version update.  Thanks for the contribution.

**Pull request merged**